ramlfications/utils.py

```diff
@@ -54,24 +54,51 @@ def load_schema(data):
     return data
 
 
-def requests_download_xml():
+def requests_download(url):
     try:
-        response = requests.get(IANA_URL)
+        response = requests.get(url)
         return response.text
     except requests.exceptions.RequestException as e:
         msg = "Error downloading XML from IANA: {0}".format(e)
         raise MediaTypeError(msg)
 
 
-def urllib_download_xml():
+def urllib_download(url):
     try:
-        response = urllib.urlopen(IANA_URL)
+        response = urllib.urlopen(url)
     except urllib.URLError as e:
         msg = "Error downloading XML from IANA: {0}".format(e)
         raise MediaTypeError(msg)
     return response.read()
 
 
+def download_url(url):
+    def setup_logger():
```

Comment:
Hmm so I think because there is a nearly the same `setup_logger` function in `update_mime_types`, I think we can pull these functions out into the module scope, and just pass in a string for the actual message, if that makes sense.